python/run_fccanalysis.py

```diff
@@ -569,7 +597,7 @@ def run_local(config: dict[str, any],
     if nevents_orig > 0:
         info_msg += f'\nReduction factor total:  {outn/nevents_orig}'
     info_msg += '\n'
-    info_msg += 80 * '='
+    info_msg += 80 * '=' 
```

Comment:
Why the space?

Reply:
Oops, no idea where that came from :D Thanks for spotting